meta/runtime.yml

```diff
@@ -296,6 +295,10 @@ plugin_routing:
       deprecation:
         removal_date: 2021-12-01
         warning_text: see plugin documentation for details
+    vmware_vmkernel_ip_config:
+      deprecation:
+        removal_date: 2023-03-01
```

Comment:
@Akasurde 
Is the date correct for removing the module?

Reply:
yes, @goneri Can you please cross-check?

Reply:
@Akasurde @goneri @sky-joker I understood that two years was the accepted deprecation period for modules. But maybe I was wrong... if you prefer a different `removal_date` just let me know.

Reply:
2 years is a lot. community.aws will reprecate a bunch of things the 2021-12-01, I suggest we do the same which would allow us to target ansible 4 ( https://docs.ansible.com/ansible/devel/roadmap/COLLECTIONS_4.html ). If we change our mind, it's easier to keep the the deprecated module a bit more longer in the collection.